src/components/organisms/EntityDetails/ExecutionDetailsDrawer/ExecutionDetailsDrawer.styled.tsx

```diff
@@ -1,17 +1,21 @@
 import {motion} from 'framer-motion';
 import styled from 'styled-components';
 
-export const ExecutionDetailsDrawerWrapper = styled(motion.div)<{$isRowSelected: boolean; drawerWidth: string}>`
+export const ExecutionDetailsDrawerWrapper = styled(motion.div)<{
+  $isRowSelected: boolean;
+  drawerWidth: string;
+  bannerVisible: boolean;
+}>`
   position: relative;
 
   display: flex;
   flex-direction: column;
 
-  height: 100%;
+  height: ${({bannerVisible}) => (bannerVisible ? 'calc(100% - 140px)' : '100%')};
```

Comment:
Aren't we able to do the height flexible, instead of controlling it that way? Probably @pavloburchak works on that though